frontend/stores/assets-store.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { RpcMethods } from '../lib/client-rpc-methods'
+import { useAssetsStore } from './assets-store'
+
+const assetsMock = [
```

Comment:
this mock is different to the data provided from the rest endpoint. Think this must have been generate through GQL.

See: https://vega-mainnet-data.commodum.io/api/v2/assets